src/app/containers/Copyright/__snapshots__/index.test.jsx.snap

```diff
@@ -74,6 +78,10 @@ exports[`should render Copyright with persian service context 1`] = `
   >
      ، منبع تصویر
   </span>
-  توصیف چیزی که اتفاق می افتد
+  <span
+    lang="en-gb"
+  >
+    توصیف چیزی که اتفاق می افتد
```

Comment:
The original text here for the Persian example is incorrect. The copyright text will always be in English, even for the Persian service. 
```suggestion
    GETTY IMAGES
```